[docs] Hide account links and use a placeholder dash (#10251)
expo/expo: docs/pages/accounts/working-together.md

title: Working Together
---

You can grant other users access to the projects belonging to your Personal Account with Expo Teams. The type of access depends on the granted role. You can [sign up for Teams](https://expo.io/dashboard/[account]/settings/services) on any of your accounts.

## Adding Members

> You can invite new members to your Personal Account, or any account you administrate, from the [Members page under Settings](https://expo.io/dashboard/[account]/settings/members). You can only add existing Expo users as a member, you can direct them to [https://expo.io/signup](https://expo.io/signup) if they don't have an account yet.

Reviewer: ```suggestion
You can invite new members to your Personal Account, or any account you administrate, from the [Members page](https://expo.io/dashboard/[account]/settings/members) in your dashboard. You can only add users with Expo accounts as members; you can direct them to [https://expo.io/signup](https://expo.io/signup) if they don't have an account yet.
```

## Removing members

> To remove members, go to the [Expo Dashboard Settings Members](https://expo.io/dashboard/[account]/settings/members) page and revoke access.

Reviewer: ```suggestion
To remove members, go to the [Members](https://expo.io/dashboard/[account]/settings/members) page and revoke access.
```
